### ISSUE TITLE
Enhancement: Add template for base16-builder

### ISSUE THREAD

**Issue by dumblob** [mawww/kakoune#65]
It would be nice to ease color themes creation e.g. by supplying (and maintaining in case highlighters and aliases will change in Kakoune) a template to repository https://github.com/chriskempson/base16-builder/tree/master/templates .

**Comment by Delapouite** [comment]
In the meantime, here's a first attempt at generating base16 colorschemes for kakoune : https://github.com/Delapouite/base16-kakoune

Improvements are welcomed.

**Comment by lenormf** [comment]
>This repository is deprecated as Base16 is changing the way it is built and maintained

Too bad.

**Comment by dumblob** [comment]
@lenormf it should be very easy to change the @Delapouite's script to the new [format and structure](https://github.com/chriskempson/base16#scheme-repositories ).

**Comment by Delapouite** [comment]
Yes, I was about to use this structure. But what I don't understand is that there used to be a faily large amount of themes before (the one I has previously gathered in base16.json). And they suddently disappear : https://github.com/chriskempson/base16-schemes-source/blob/master/list.yaml https://github.com/chriskempson/base16-unclaimed-schemes

I can understand that Chris Kempson do not have time to handle everything anymore, but exploding his community in a myriad of independant repositories make things hard to follow. If they were all centralized in a github bas16 organisation managed by all this wonderful people it would be less cumbersome.

**Comment by leira** [comment]
I just made a base 16 template https://github.com/leira/base16-kakoune. Then I found @Delapouite already did a version... Sorry, should have searched here first.

My repo is using the new builder structure. I will check @Delapouite's repo, to see if I can steal any thing.

**Comment by lenormf** [comment]
I think this issue can be closed.

**Comment by dumblob** [comment]
Right.

**Comment by Vintodrimmer** [comment]
Is there any guide on how to convert` Vim` theme to `Kakoune`?

**Comment by Delapouite** [comment]
Hi. Not there is not. But you can take some inspiration from this repo : https://github.com/delapouite/kakoune-colors
